base/deprecated.jl

```diff
@@ -1708,6 +1708,9 @@ export hex2num
 # PR 23341
 @deprecate diagm(A::SparseMatrixCSC) spdiagm(sparsevec(A))
 
+# PR #23393
+@deprecate titlecase(s::AbstractString) titlecase(s, false, true)
```

Comment:
This is tricky; we don't want to tell people to call the 3-argument version in their code.